comps/comps-foreman-rhel6.xml

```diff
@@ -270,8 +270,8 @@
       <packagereq type="default">tfm-rubygem-hirb-doc</packagereq>
       <packagereq type="default">tfm-rubygem-hirb-unicode-doc</packagereq>
       <packagereq type="default">tfm-rubygem-hoe-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-i18n-doc</packagereq>
       <packagereq type="default">tfm-rubygem-ipaddress-doc</packagereq>
+      <packagereq type="default">tfm-rubygem-i18n-doc</packagereq>
```

Comment:
I think this is a difference between en_GB and cs_CZ locales/collation... I'd appreciate it if we could keep the order to maintain our sanity.  Perhaps set LC_COLLATE=C in comp_docs.sh?